modules/tinymce/src/core/test/ts/browser/textpatterns/FindInlinePatternTest.ts

```diff
@@ -326,7 +333,7 @@ describe('browser.tinymce.textpatterns.FindInlinePatternTest', () => {
               start: '*', end: '*', format: 'bold'
             }
           ];
-        } else if (parentTag === 'div' && ctx.text === 'replace-me') {
+        } else if (parentTag === 'div' && ctx.text.includes('replace-me')) {
```

Comment:
This won't test what you want properly for the `Inline pattern matches text in the middle` test... because this would still match if you passed the entire block text.

- [ ] Add or fix the test to that ensures `replace-me` (or similar) isn't returned if it's after the trigger point

Reply:
Have changed the logic and added another test